tests/components/image_processing/test_facebox.py

```diff
@@ -147,6 +162,61 @@ def mock_face_event(event):
     assert state.attributes.get('matched_faces') == {}
 
 
+@patch('os.access', Mock(return_value=True))
+@patch('os.path.isfile', Mock(return_value=True))
+async def test_teach_service(hass, mock_image):
```

Comment:
This test is never awaited. Please run the tests locally, eg with `script/lazytox.py`. You'll see the warning in the pytest warning summary.